### ISSUE TITLE
[Bug]: Treesitter not showing up as a source since commit 35988af

### ISSUE THREAD

**Issue by tombakas** [Bekaboo/dropbar.nvim#189]
### Description

Breadcrumbs are not displayed if TreeSitter is used as the only source. I traced it down to commit 35988af. If the commit before it is used, breadcrumbs show up as expected.

### nvim version

v0.10.2

### dropbar.nvim version

9d8cad9548804a33c9a1968fe305e46e95c911ae

### Operating system and version

macOS 14.6.1

### Minimal config

```Lua
I didn't minimize the config, as I am fairly sure the rest of the config did not interfere with the functioning of the plugin.
```

### Steps to reproduce

Specify `treesitter` as the only source in the config and open a file type that you have treesitter installed for.

### Expected behavior

Breadcrumbs are visible

### Actual behavior

No breadcrumbs

**Comment by Bekaboo** [comment]
Should be fixed in commit ef933aa. Does it work for you?

**Comment by tombakas** [comment]
Yes, this solves the issue, thank you.
